test/Casbin.AspNetCore.Performance/Casbin.AspNetCore.Performance.csproj

```diff
@@ -2,20 +2,39 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFrameworks>net5;netcoreapp3.1</TargetFrameworks>
+    <TargetFrameworks>net6;net5;netcoreapp3.1</TargetFrameworks>
```

Comment:
11 min is too long

![image](https://user-images.githubusercontent.com/3787410/186382058-c27590b5-5417-4304-9e14-335d3b4d976f.png)